src/ui/components/filters/filterboxcomponent.js

```diff
@@ -102,7 +101,7 @@ class FilterBoxConfig {
      * The list of filters to display and control, ignoring empty sections
      * @type {object[]}
      */
-    this.filterConfigs = config.filters.filter(f => f.options.length !== 0);
```

Comment:
This line makes me think FilterBox was only ever meant to support FilterOptions

Reply:
You can hack either DateRangeFilter or RangeFilter into FilterBox by specifying options: [{}] in the config, though, and they will work as expected (though maybe the fact that they work wasn't expected?)